main.py

```diff
@@ -180,7 +181,7 @@ def import_streams_from_clipboard(self):
         for stream in streams:
             self.add_new_stream(stream_url=stream)
 
-    def add_new_stream(self, stream_url=None, stream_quality=None):
+    def add_new_stream(self, stream_url=None, stream_qualities=cfg[CONFIG_QUALITY]):
```

Comment:
This forces a restar. In my opinion it is better to have it dynamically use whatever is in the config.